iOS/docs/kickoff/configure-ci/README.md

```diff
@@ -0,0 +1,171 @@
+
+![choose project](./resources/br2.png)
+
+### Setup repository access
+
+Select `auto-add SSH Key`. The key should be added automatically after a moment. If it doesn’t work, you may need to do it manually.
+
+NOTE: Only Github admins can automatically add the SSH key in this step. In case you have no access, skip this step and ask your TL to do it.
+
+![choose project](./resources/br3.png)
+
+### Validation setup
+
+Choose the main branch of your project (master or development usually). Bitrise will scan the branch to check for the correct configurations. This takes about 5 minutes.
+
+### Project build configuration
+
+Select iOS. Verify that the correct XCode stack is selected and confirm.
+
+![choose project](./resources/br4.png)
+
+### Webhook setup
+
+Select Register a new webhook for me. It will allow Bitrise to know when a `push` / `pull request` has been made, so it can test it. This step can only be done by an admin.
+
+NOTE: Only Github admins can automatically add the webhook in this step. In case you have no access, skip this step and ask your TL to do it. It can be done from the Code tab in the project’s dashboard. Under “webhooks”, there’s a webhook URL which you can add to your Github repository following [this guide](https://github.com/bitrise-io/bitrise-webhooks#github---setup--usage).
+
+### Finish configuration
+
+Select We’ve kicked off your first test build for you to continue. Bitrise will run a build at this step, you should cancel it since it runs with another configuration, thus it will fail.
+
+![choose project](./resources/br5.png)
+
+## Configure Bitrise’s `bitrise.yml`
+
+When Bitrise builds trigger, they look for the configuration in the web’s `bitrise.yml`. That file will call the one from our repository. 
+
+For this, go to `Workflow`, inside the project's bitrise page. There select `bitrise.yml` and replace the contect of the file for this:
+
+```
+format_version: 1.4.0
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+trigger_map:
+- push_branch: master
+  workflow: ci
+- pull_request_source_branch: "*"
+  pull_request_target_branch: "*"
+  workflow: ci
+workflows:
+  _run_from_repo:
+    steps:
+    - activate-ssh-key@3.1.1:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@3.4.1: {}
+    - script@1.1.3:
+        title: Extracting Bitrise configuration from repo
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            bitrise run "${BITRISE_TRIGGERED_WORKFLOW_ID}"
+  ci:
+    after_run:
+    - _run_from_repo
+    steps: 
+```
+
+## Configure App Environment variables
+
+From the left menu, select `App Env Vars` for private repositories or `Secret Env Vars` for public ones. Add the keys and values of the environment variables that will be needed for bootstrapping, testing and all other actions or steps that will be run according to the repo's `bitrise.yml`.
```

Comment:
From the menu on the left.

Reasoning: 
- Sounds better if there is no right menu (to me).